api/projects.py

```diff
@@ -119,7 +119,7 @@ def get(self, uid=None, gid=None):
             if not group:
                 self.abort(400, 'invalid group id')
         query = {'group': gid} if gid else {}
-        projection = ['group', 'name', 'notes', 'timestamp', 'timezone']
+        projection = ['group', 'name', 'notes', 'timestamp', 'timezone', 'public']
```

Comment:
Yes. The request was to be able to retrieve *permitted* and *public* projects in a single request and, more importantly, be able to access public data, despite sending an auth token. The first part is arguable, but the second part was just plain wrong before.